modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/action/FDSBulkActionList.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.action;
+
+import com.liferay.frontend.data.set.FDSEntryItemImportPolicy;
+import com.liferay.frontend.data.set.model.FDSActionDropdownItem;
+
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author Daniel Sanz
+ */
+public interface FDSBulkActionList {
+
+	public List<FDSActionDropdownItem> getFDSActionDropdownItems(
+		HttpServletRequest httpServletRequest);
+
+	public default FDSEntryItemImportPolicy getFDSEntryItemImportPolicy() {
+		return FDSEntryItemImportPolicy.ITEM_PROXY;
```

Comment:
So, a policy is defined the level of SDS list? This means all the items here will be item proxies? There is no for devs to say way to say: make this one action proxy and import others detached, as let's say link actions? Just asking out of curiosity, I'm not saying we should implement this immediately.

Reply:
Yep, that's the idea. Here, I've just mimicked current impl for creation menu and item action dropdown providers, but for bulk actions. 

Both of these providers return the entire list of actions. So, as long as computation of such list requires some sort of dynamism (even just for one action out of n), all of them must be considered as "dynamic" because current APIs are not enough fine-grained to declare that "dynamism" on a per-action basis.

Current import policy semantics capture the level of dynamism as deep as it can be at this moment, namely:

- `DETACHED` describes a 100% static list of static actions, where:
  - List of actions is always the same, does not depend on conditions resolved in backend, such as FFs
  - No constituent of any action changes over time: icon, URL, etc remain the same
  - Note frontend logic can still apply, namely: permissionKey and item props interpolation
- `ITEM_PROXY` describes a static list of actions, where
  - List of actions is always the same, does not depend on conditions resolved in backend, such as FFs
  - Constituent of some action can change over time. Normally, the URL. So, actions will be computed on every render
  - Note current APIs don't have expressive power to denote which specific actions will change and which ones will not
- `GROUP_PROXY` describes a dynamic list of actions, where
  - List of actions is computed on every render. For example, when creation menu includes items depending on FFs  or on a dynamic set of data (such as creation of a web content); or item actions are computed by an action provider in the classic display
  - Constituent of some action can change over time, but that's not relevant because the entire list is computed on every render

Implementing this policy on an action level looks feasible but needs more involvement on action implementors and be captured on existing APIs to define them, for example as extra method returning an array of action ids along with the policies for each. Most likely these could be non-breaking additions